mit-bitcoin-expo/mit-bitcoin-expo-2022/lightning-panel.md

```diff
@@ -12,20 +12,20 @@ date: 2022-07-05
 Jeremy Rubin: 00:00:07
 
 I was hoping that we can have our panelists, and maybe we'll start with Rene.
-Just introduce themselves quickly, and also answer the question, What is Lightning to you?
+Just introduce themselves quickly, and also answer the question, what is `Lightning` to you?
 And I mean that sort of in the long run.
 Lightning is a thing today, but what is the philosophical Lightning to you?
 
 Rene Pickhardt: 00:00:26
 
-So I'm Rene Pickard, a researcher and developer for the Lightning Network.
-And for me personally, the Lightning Network is the means to really create this peer-to-peer electronic cash system so to use Bitcoin in a decentralized way and fast way.
+So I'm Rene Pickardt, a researcher and developer for the `Lightning Network`.
+And for me personally, the `Lightning Network` is the means to really create this peer-to-peer electronic cash system so to use Bitcoin in a decentralized way and fast way.
 
 Lisa Neigut: 00:00:45
 
-Cool, hey everyone I am Lisa Neigut I work at Blockstream on Core Lightning, which is a implementation of Lightning.
-To me, I guess, yeah, so I see Lightning as probably, We call it Bitcoin scaling, like the thing that's gonna scale Bitcoin.
-I'm gonna like totally get this wrong.
+Cool, hey everyone I am Lisa Neigut. I work at Blockstream on Core Lightning, which is a implementation of Lightning.
```

Comment:
We want to eventually automate this, but for now, when you are creating new sentences (By adding a period "."), try to follow the one-sentence-per-line format.